src/index.html

```diff
@@ -6,13 +6,52 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="card">
+
+        <img
+          src="images/imac.jpeg"
+          alt="iMac photo"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/31e8d342-157a-4ff5-989a-26c8ee2e8e78)